[Cross Cluster Replication] NP Shim

jloleysens opened the pull request:
## Summary

Initial shimming of Cross Cluster Replication.

### Future Work (for move out of legacy)

- Proper handling of mounting the react application
- Proper handling of license data and license route protection

### Checklist

- [ ] [Fix unit or functional tests](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility) were updated or added to match the most common scenarios

### For maintainers

- [ ] This was checked for breaking API changes and was [labeled appropriately](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#release-notes-process)

elasticmachine commented:
Pinging @elastic/es-ui (Team:Elasticsearch UI)

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/)
* Commit: 7d8d0cc77d138616269c23e8ac670bd465039e5e

---

### [Test Failures](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/testReport)

<details><summary>Kibana Pipeline / x-pack-intake-agent / X-Pack Jest Tests.x-pack/legacy/plugins/cross_cluster_replication/__jest__/client_integration.<AutoFollowPatternList /> when there are no auto-follow patterns should display an empty prompt</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/testReport//junit/X-Pack%20Jest%20Tests/x-pack_legacy_plugins_cross_cluster_replication___jest___client_integration/Kibana_Pipeline___x_pack_intake_agent____AutoFollowPatternList____when_there_are_no_auto_follow_patterns_should_display_an_empty_prompt)


#### Standard Out
```
Failed Tests Reporter:
  - Test has not failed recently on tracked branches


```

#### Stack Trace
```
Error: expect(received).toBe(expected) // Object.is equality

Expected: true
Received: false
    at Object.test (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/x-pack/legacy/plugins/cross_cluster_replication/__jest__/client_integration/auto_follow_pattern_list.test.js:67:37)
    at Object.asyncJestTest (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at resolve (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/queueRunner.js:43:12)
    at new Promise (<anonymous>)
    at mapper (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at promise.then (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/queueRunner.js:73:41)
```
</details>

---

<details><summary>Kibana Pipeline / x-pack-intake-agent / X-Pack Jest Tests.x-pack/legacy/plugins/cross_cluster_replication/__jest__/client_integration.<AutoFollowPatternList /> when there are no auto-follow patterns should have a button to create a follower index</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/testReport//junit/X-Pack%20Jest%20Tests/x-pack_legacy_plugins_cross_cluster_replication___jest___client_integration/Kibana_Pipeline___x_pack_intake_agent____AutoFollowPatternList____when_there_are_no_auto_follow_patterns_should_have_a_button_to_create_a_follower_index)


#### Standard Out
```
Failed Tests Reporter:
  - Test has not failed recently on tracked branches


```

#### Stack Trace
```
Error: expect(received).toBe(expected) // Object.is equality

Expected: true
Received: false
    at Object.test (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/x-pack/legacy/plugins/cross_cluster_replication/__jest__/client_integration/auto_follow_pattern_list.test.js:71:55)
    at Object.asyncJestTest (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at resolve (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/queueRunner.js:43:12)
    at new Promise (<anonymous>)
    at mapper (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at promise.then (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/node_modules/jest-jasmine2/build/queueRunner.js:73:41)
```
</details>

---

<details><summary>Kibana Pipeline / x-pack-intake-agent / X-Pack Jest Tests.x-pack/legacy/plugins/cross_cluster_replication/__jest__/client_integration.<AutoFollowPatternList /> when there are auto-follow patterns should not display the empty prompt</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/testReport//junit/X-Pack%20Jest%20Tests/x-pack_legacy_plugins_cross_cluster_replication___jest___client_integration/Kibana_Pipeline___x_pack_intake_agent____AutoFollowPatternList____when_there_are_auto_follow_patterns_should_not_display_the_empty_prompt)


#### Standard Out
```
Failed Tests Reporter:
  - Test has not failed recently on tracked branches


```

#### Stack Trace
```
Error: Eui Table "autoFollowPatternListTable" not found.
    at Object.getMetaData (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/x-pack/test_utils/testbed/testbed.ts:218:17)
    at Object.beforeEach (/var/lib/jenkins/workspace/elastic+kibana+pipeline-pull-request/kibana/x-pack/legacy/plugins/cross_cluster_replication/__jest__/client_integration/auto_follow_pattern_list.test.js:110:43)
```
</details>

---
and 116 more failures, only showing the first 3.

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/","number":"33106","commit":"7d8d0cc77d138616269c23e8ac670bd465039e5e"}]}
PIPELINE-->

jloleysens commented:
@elasticmachine merge upstream

jloleysens commented on x-pack/legacy/plugins/cross_cluster_replication/public/np_ready/app/components/auto_follow_pattern_form.js:
```diff
@@ -29,12 +29,12 @@ import {
 import { extractQueryParams } from '../services/query_params';
 import { getRemoteClusterName } from '../services/get_remote_cluster_name';
-import { API_STATUS } from '../constants';
+import { API_STATUS, INDEX_ILLEGAL_CHARACTERS_VISIBLE } from '../constants';
```
Yeah I saw this guy coming up! I'll resolve once it is merged.

jloleysens commented on x-pack/legacy/plugins/cross_cluster_replication/public/np_ready/app/services/breadcrumbs.ts:
```diff
@@ -3,9 +3,27 @@
+import { ManagementAppMountParams } from '../../../../../../../../src/plugins/management/public';
+
+// @ts-ignore
+import { BASE_PATH } from '../../../../common/constants';
```
Nope! Placeholder from when it was a JS file!

jloleysens commented on x-pack/legacy/plugins/cross_cluster_replication/public/np_ready/app/services/documentation_links.ts:
```diff
@@ -0,0 +1,22 @@
+  esBase = `${ELASTIC_WEBSITE_URL}guide/en/elasticsearch/reference/${DOC_LINK_VERSION}`;
+};
+
+export const autoFollowPatternUrl = () => `${esBase}/ccr-put-auto-follow-pattern.html`;
```
Great point!

jloleysens commented:
@alisonelizabeth Thanks for taking a look!

> I cannot edit a follower index and get the following error:

Hmm, curious, I remember seeing this on the create and then I changed the request body to be a string. Perhaps I hadn't pushed that change - I am able to create and edit auto follow patterns and create follower indices (but not edit them, it looks disallowed by the UI). Would you mind taking another look?

> It looks like the Follower badge is missing in Index Management...

Very well caught! It looks like I introduced a logic regression for the data enricher on the server side! Just pushed a fix :+1:

jloleysens commented on x-pack/legacy/plugins/cross_cluster_replication/public/np_ready/app/components/auto_follow_pattern_action_menu/auto_follow_pattern_action_menu.container.ts:
```diff
@@ -10,7 +10,7 @@ import {
 
 // @ts-ignore
-import { pauseAutoFollowPattern, resumeAutoFollowPattern } from '../../store/actions';
+import { pauseAutoFollowPattern, resumeAutoFollowPattern } from '../../store/actions/index';
```
And by :+1: I mean no of course 😄

alisonelizabeth commented:
@jloleysens thanks for fixing the issue with the badge, looks good now!

> I am able to create and edit auto follow patterns and create follower indices (but not edit them, it looks disallowed by the UI).

What do you mean by "it looks disallowed"? Maybe we have something set up differently? It is possible on my end, and I am still seeing the same error that I mentioned above. As you said, create/edit auto-follow patterns works as expected.

<img width="1276" alt="Screen Shot 2020-03-16 at 1 02 05 PM" src="https://user-images.githubusercontent.com/5226211/76782803-38d5c100-6787-11ea-9b23-4b55d82cfd72.png">
<img width="1795" alt="Screen Shot 2020-03-16 at 1 03 33 PM" src="https://user-images.githubusercontent.com/5226211/76782806-396e5780-6787-11ea-866b-08d05ac3fabb.png">

jloleysens commented:
@alisonelizabeth Ah ok I see, I was referring to the the greyed out text fields, but I see there is an advanced section that can be edited.

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33496/)
* Commit: 390dc6b30048dac6080883878c06d2e5b8b8d2ec

### History
* :broken_heart: [Build #33106](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/) failed 7d8d0cc77d138616269c23e8ac670bd465039e5e

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/","number":"33106","commit":"7d8d0cc77d138616269c23e8ac670bd465039e5e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33496/","number":"33496","commit":"390dc6b30048dac6080883878c06d2e5b8b8d2ec"}]}
PIPELINE-->

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33542/)
* Commit: 49631db124e18104adf3121e49ac30fa8d73a26f

### History
* :broken_heart: [Build #33496](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33496/) failed 390dc6b30048dac6080883878c06d2e5b8b8d2ec
* :broken_heart: [Build #33106](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/) failed 7d8d0cc77d138616269c23e8ac670bd465039e5e

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/","number":"33106","commit":"7d8d0cc77d138616269c23e8ac670bd465039e5e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33496/","number":"33496","commit":"390dc6b30048dac6080883878c06d2e5b8b8d2ec"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33542/","number":"33542","commit":"49631db124e18104adf3121e49ac30fa8d73a26f"}]}
PIPELINE-->

jloleysens commented:
@elasticmachine merge upstream

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33707/)
* Commit: 50430e70330792271f41fb44281266ab60eb335f

### History
* :broken_heart: [Build #33542](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33542/) failed 49631db124e18104adf3121e49ac30fa8d73a26f
* :broken_heart: [Build #33496](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33496/) failed 390dc6b30048dac6080883878c06d2e5b8b8d2ec
* :broken_heart: [Build #33106](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/) failed 7d8d0cc77d138616269c23e8ac670bd465039e5e

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/","number":"33106","commit":"7d8d0cc77d138616269c23e8ac670bd465039e5e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33496/","number":"33496","commit":"390dc6b30048dac6080883878c06d2e5b8b8d2ec"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33542/","number":"33542","commit":"49631db124e18104adf3121e49ac30fa8d73a26f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33707/","number":"33707","commit":"50430e70330792271f41fb44281266ab60eb335f"}]}
PIPELINE-->

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33716/)
* Commit: a38eeb40ee707bb4032fb6b34768af26f2fe0b48

---

### [Test Failures](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33716/testReport)

<details><summary>Kibana Pipeline / kibana-intake-agent / Server Mocha Tests.x-pack/legacy/plugins/cross_cluster_replication/server/np_ready/lib/license_pre_routing_factory/__tests__/license_pre_routing_factory·js.license_pre_routing_factory #reportingFeaturePreRoutingFactory only instantiates one instance per server</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33716/testReport//junit/Server%20Mocha%20Tests/x-pack_legacy_plugins_cross_cluster_replication_server_np_ready_lib_license_pre_routing_factory___tests___license_pre_routing_factory·js/Kibana_Pipeline___kibana_intake_agent___license_pre_routing_factory__reportingFeaturePreRoutingFactory_only_instantiates_one_instance_per_server)


#### Standard Out
```
Failed Tests Reporter:
  - Test has not failed recently on tracked branches


```

#### Stack Trace
```
TypeError: Cannot read property 'server' of undefined
    at licensePreRoutingFactory (x-pack/legacy/plugins/cross_cluster_replication/server/np_ready/lib/license_pre_routing_factory/license_pre_routing_factory.ts:17:36)
    at Context.it (x-pack/legacy/plugins/cross_cluster_replication/server/np_ready/lib/license_pre_routing_factory/__tests__/license_pre_routing_factory.js:30:29)
```
</details>

---

<details><summary>Kibana Pipeline / kibana-intake-agent / Server Mocha Tests.x-pack/legacy/plugins/cross_cluster_replication/server/np_ready/lib/license_pre_routing_factory/__tests__/license_pre_routing_factory·js.license_pre_routing_factory #reportingFeaturePreRoutingFactory isAvailable is false replies with 403</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33716/testReport//junit/Server%20Mocha%20Tests/x-pack_legacy_plugins_cross_cluster_replication_server_np_ready_lib_license_pre_routing_factory___tests___license_pre_routing_factory·js/Kibana_Pipeline___kibana_intake_agent___license_pre_routing_factory__reportingFeaturePreRoutingFactory_isAvailable_is_false_replies_with_403)


#### Standard Out
```
Failed Tests Reporter:
  - Test has not failed recently on tracked branches


```

#### Stack Trace
```
TypeError: Cannot read property 'server' of undefined
    at licensePreRoutingFactory (x-pack/legacy/plugins/cross_cluster_replication/server/np_ready/lib/license_pre_routing_factory/license_pre_routing_factory.ts:17:36)
    at Context.it (x-pack/legacy/plugins/cross_cluster_replication/server/np_ready/lib/license_pre_routing_factory/__tests__/license_pre_routing_factory.js:44:35)
```
</details>

---

<details><summary>Kibana Pipeline / kibana-intake-agent / Server Mocha Tests.x-pack/legacy/plugins/cross_cluster_replication/server/np_ready/lib/license_pre_routing_factory/__tests__/license_pre_routing_factory·js.license_pre_routing_factory #reportingFeaturePreRoutingFactory isAvailable is true replies with nothing</summary>

[Link to Jenkins](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33716/testReport//junit/Server%20Mocha%20Tests/x-pack_legacy_plugins_cross_cluster_replication_server_np_ready_lib_license_pre_routing_factory___tests___license_pre_routing_factory·js/Kibana_Pipeline___kibana_intake_agent___license_pre_routing_factory__reportingFeaturePreRoutingFactory_isAvailable_is_true_replies_with_nothing)


#### Standard Out
```
Failed Tests Reporter:
  - Test has not failed recently on tracked branches


```

#### Stack Trace
```
TypeError: Cannot read property 'server' of undefined
    at licensePreRoutingFactory (x-pack/legacy/plugins/cross_cluster_replication/server/np_ready/lib/license_pre_routing_factory/license_pre_routing_factory.ts:17:36)
    at Context.it (x-pack/legacy/plugins/cross_cluster_replication/server/np_ready/lib/license_pre_routing_factory/__tests__/license_pre_routing_factory.js:60:35)
```
</details>

---

### History
* :broken_heart: [Build #33707](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33707/) failed 50430e70330792271f41fb44281266ab60eb335f
* :broken_heart: [Build #33542](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33542/) failed 49631db124e18104adf3121e49ac30fa8d73a26f
* :broken_heart: [Build #33496](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33496/) failed 390dc6b30048dac6080883878c06d2e5b8b8d2ec
* :broken_heart: [Build #33106](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/) failed 7d8d0cc77d138616269c23e8ac670bd465039e5e

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/","number":"33106","commit":"7d8d0cc77d138616269c23e8ac670bd465039e5e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33496/","number":"33496","commit":"390dc6b30048dac6080883878c06d2e5b8b8d2ec"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33542/","number":"33542","commit":"49631db124e18104adf3121e49ac30fa8d73a26f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33707/","number":"33707","commit":"50430e70330792271f41fb44281266ab60eb335f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33716/","number":"33716","commit":"a38eeb40ee707bb4032fb6b34768af26f2fe0b48"}]}
PIPELINE-->

kibanamachine commented:
## :broken_heart: Build Failed
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33749/)
* Commit: ca2cd89b9d5fa28354db6d0ba47741a0323f8d7e

### History
* :broken_heart: [Build #33716](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33716/) failed a38eeb40ee707bb4032fb6b34768af26f2fe0b48
* :broken_heart: [Build #33707](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33707/) failed 50430e70330792271f41fb44281266ab60eb335f
* :broken_heart: [Build #33542](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33542/) failed 49631db124e18104adf3121e49ac30fa8d73a26f
* :broken_heart: [Build #33496](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33496/) failed 390dc6b30048dac6080883878c06d2e5b8b8d2ec
* :broken_heart: [Build #33106](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/) failed 7d8d0cc77d138616269c23e8ac670bd465039e5e

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33106/","number":"33106","commit":"7d8d0cc77d138616269c23e8ac670bd465039e5e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33496/","number":"33496","commit":"390dc6b30048dac6080883878c06d2e5b8b8d2ec"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33542/","number":"33542","commit":"49631db124e18104adf3121e49ac30fa8d73a26f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33707/","number":"33707","commit":"50430e70330792271f41fb44281266ab60eb335f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33716/","number":"33716","commit":"a38eeb40ee707bb4032fb6b34768af26f2fe0b48"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33749/","number":"33749","commit":"ca2cd89b9d5fa28354db6d0ba47741a0323f8d7e"}]}
PIPELINE-->

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33830/)
* Commit: 3cbb917bd76d2cb8b3f9273a0e1893f6b6e4660d

### History
* :broken_heart: [Build #33749](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33749/) failed ca2cd89b9d5fa28354db6d0ba47741a0323f8d7e
* :broken_heart: [Build #33716](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33716/) failed a38eeb40ee707bb4032fb6b34768af26f2fe0b48
* :broken_heart: [Build #33707](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33707/) failed 50430e70330792271f41fb44281266ab60eb335f
* :broken_heart: [Build #33542](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33542/) failed 49631db124e18104adf3121e49ac30fa8d73a26f
* :broken_heart: [Build #33496](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33496/) failed 390dc6b30048dac6080883878c06d2e5b8b8d2ec

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33496/","number":"33496","commit":"390dc6b30048dac6080883878c06d2e5b8b8d2ec"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33542/","number":"33542","commit":"49631db124e18104adf3121e49ac30fa8d73a26f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33707/","number":"33707","commit":"50430e70330792271f41fb44281266ab60eb335f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33716/","number":"33716","commit":"a38eeb40ee707bb4032fb6b34768af26f2fe0b48"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33749/","number":"33749","commit":"ca2cd89b9d5fa28354db6d0ba47741a0323f8d7e"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33830/","number":"33830","commit":"3cbb917bd76d2cb8b3f9273a0e1893f6b6e4660d"}]}
PIPELINE-->

jloleysens commented:
Waiting for https://github.com/elastic/kibana/pull/60186 to merge, will need to clean up duplication of ui/indices code

cjcenizal commented:
@jloleysens Just merged https://github.com/elastic/kibana/pull/60186

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/34507/)
* Commit: b4bd35f41eb54c22e6c1735640e42d14ed4eebb6

### History
* :green_heart: [Build #33830](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33830/) succeeded 3cbb917bd76d2cb8b3f9273a0e1893f6b6e4660d
* :broken_heart: [Build #33749](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33749/) failed ca2cd89b9d5fa28354db6d0ba47741a0323f8d7e
* :broken_heart: [Build #33716](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33716/) failed a38eeb40ee707bb4032fb6b34768af26f2fe0b48
* :broken_heart: [Build #33707](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33707/) failed 50430e70330792271f41fb44281266ab60eb335f
* :broken_heart: [Build #33542](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33542/) failed 49631db124e18104adf3121e49ac30fa8d73a26f

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33542/","number":"33542","commit":"49631db124e18104adf3121e49ac30fa8d73a26f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33707/","number":"33707","commit":"50430e70330792271f41fb44281266ab60eb335f"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33716/","number":"33716","commit":"a38eeb40ee707bb4032fb6b34768af26f2fe0b48"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33749/","number":"33749","commit":"ca2cd89b9d5fa28354db6d0ba47741a0323f8d7e"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/33830/","number":"33830","commit":"3cbb917bd76d2cb8b3f9273a0e1893f6b6e4660d"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/34507/","number":"34507","commit":"b4bd35f41eb54c22e6c1735640e42d14ed4eebb6"}]}
PIPELINE-->
